docusaurus.config.js

```diff
@@ -131,7 +131,19 @@ module.exports = {
             to: '/docs/archive/working_with_teams'
           }
         ],
-        
+        createRedirects(existingPath) {
+          // For every canonical page under /docs/components/EFSPIntegration,
+          // create a redirect-from path under /docs/efiling so old URLs
+          // like /docs/efiling/... redirect to the new canonical pages.
+          if (existingPath.startsWith('/docs/components/EFSPIntegration')) {
+            const subPath = existingPath.substring('/docs/components/EFSPIntegration'.length);
+            // include both base and subpaths
+            if (subPath === '' || subPath.startsWith('/')) {
```

Comment:
FWIW, the case where `subpath === ''`, (i.e on https://assemblyline.suffolklitlab.org/docs/components/EFSPIntegration) doesn't currently exist, and I don't think that it's going to be added given there's already an overview page at `.../overview`.